EnglishTranslation-master/ITEM.tsv

```diff
@@ -689,162 +689,162 @@ ITEM_20150317_000688	Breach Pike
 ITEM_20150317_000810	Serpens 
-ITEM_20150317_000811	Users of this wand must be able to control their own darkness within.
+ITEM_20150317_000811	Users of this wand must be able to control their own darkness within. 
 ITEM_20150317_000812	Levereuse 
-ITEM_20150317_000813	People with strong mind and self-realization can utilize Levereuse to its full potential during battle.
+ITEM_20150317_000813	People with strong mind and self-realization can utilize Levereuse to its full potential during battle. 
 ITEM_20150317_000814	Kyrnus 
-ITEM_20150317_000815	Prior to the Great Plant Cataclym, there was nobody who would use this weapon because it used demonic powers. However, with the vanishing of the goddesses, desperate people have begun using it.
+ITEM_20150317_000815	Prior to the Great Plant Cataclym, there was nobody who would use this weapon because it used demonic powers. However, with the vanishing of the goddesses, desperate people have begun using it. 
 ITEM_20150317_000816	Knife 
 ITEM_20150317_000817	Sub-weapon 
-ITEM_20150317_000818	It is a small knife used for various purposes such as battle, cooking, or camping.
+ITEM_20150317_000818	It is a small knife used for various purposes such as battle, cooking, or camping. 
 ITEM_20150317_000819	Dirk Dagger
 ITEM_20150317_000820	Stronger than normal knives. Some use it for throwing. 
 ITEM_20150317_000821	Main Gauche 
-ITEM_20150317_000822	Used as a support weapon but very capable of defending or deflecting enemy attacks.
-ITEM_20150317_000823	A short dagger with double edge. It's an evolved form of the tool knife used by the Highlanders with an elongated blade.
-ITEM_20150317_000824	A dagger made to be easily used left-handed. Designed with the purpose of blocking an opponent's sword by using the Guard skill. The grip is mirrored so that it fits well to your left hand.
+ITEM_20150317_000822	Used as a support weapon but very capable of defending or deflecting enemy attacks. 
+ITEM_20150317_000823	A short dagger with double edge. It's an evolved form of the tool knife used by the Highlanders with an elongated blade. 
+ITEM_20150317_000824	A dagger made to be easily used left-handed. Designed with the purpose of blocking an opponent's sword by using the Guard skill. The grip is mirrored so that it fits well to your left hand. 
 ITEM_20150317_000825	Chura 
-ITEM_20150317_000826	Single-edged, sharply pointed dagger. A specialized design for stabbing.
+ITEM_20150317_000826	Single-edged, sharply pointed dagger. Specialized design for stabbing. 
 ITEM_20150317_000827	Pescas 
-ITEM_20150317_000828	Single-edge dagger with extremely sharp, long thin tip. It has an 'S' shape body with a heavy, delicately decorated hilt.
+ITEM_20150317_000828	Single-edge dagger with extremely sharp, long thin tip. It has a 'S' shape body with a heavy, delicately decorated hilt. 
```

Comment:
Bluh, notepad+ or pasting it back or SOMETHING did some autocorrecting.  Thanks for the catching this and the other thing.  I didn't even read the lines outside of the ones listed in my main comment.